WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -246,7 +246,7 @@ private static void CurrentDomain_UnhandledException(object? sender, UnhandledEx
 	}
 
 	[SuppressMessage("CodeQuality", "IDE0051:Remove unused private members", Justification = "Required to bootstrap Avalonia's Visual Previewer")]
-	private static AppBuilder BuildAvaloniaApp() => AppBuilder.Configure(() => new App()).UseReactiveUI().SetupAppBuilder();
+	private static AppBuilder BuildAvaloniaApp() => AppBuilder.Configure(() => new App()).UseReactiveUI().UsePlatformDetect();
```

Comment:
The last I heard in the past is that the designer should use the same setup. Are you sure about this?

Reply:
I don't have the knowledge to decide. @wieslawsoltes @jmacato your call.

Reply:
> I don't have the knowledge to decide. @wieslawsoltes @jmacato your call.

Me neither. I was being pragmatic this time 🙂

Reply:
If you add here https://github.com/zkSNACKs/WalletWasabi/blob/473d368173d0f7f80ea97348af40c8954e80e540/WalletWasabi.Fluent.Desktop/Extensions/AppBuilderExtension.cs#L16 `.UsePlatformDetect()` does it solve the issue ?